client/reader/sidebar/sidebarHOC.jsx

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import { savePreference } from 'calypso/state/preferences/actions';
+import { getSidebarIsCollapsed } from 'calypso/state/ui/selectors';
+
+export const sidebarHOC = ( Component ) => {
```

Comment:
Naming nit: We generally name high-order components `withSomething`, so this could be named `withAlwaysExpandedSidebar` or something similar. Having HOC in the name feels redundant.

Reply:
Reverted this commit. Though, HOC is great learning for me.